client/src/index.jsx

```diff
@@ -17,5 +17,5 @@ render((
     <Route path="/login" component={Login} />
     <Route path="/logout" component={Logout} onEnter={requireAuth} />
   </Router>
-  ), document.getElementById('app')
+  ), document.getElementById('app'),
```

Comment:
This is from `comma-dangle` for functions, as [here](http://eslint.org/docs/rules/comma-dangle#functions).